### PR TITLE
Fix links

### PR DESCRIPTION
**Pull request by sandcha**
**Bug fix :**

- [x] Fixes documentation link in index page body (https → http)
- [x] Fixes links colour in cookies page
- [x] Adds a CHANGELOG file

**Before :**

capture d ecran 2018-03-15 a 14 42 58

**After :**

capture d ecran 2018-03-15 a 14 43 30

### CODE REVIEW

File: pages/cookies.js [hunk 1]
```diff
@@ -56,7 +56,12 @@ const Cookies = () => (
         font-weight: 300; 
       }
 
-     @media (max-width: 720px) {
+      .reference {
```
Comment (maukoquiroga) [inline pages/cookies.js]: Shouldn't it just be ```a {```?

Comment (sandcha) [reply pages/cookies.js]: Fixed in `78a66c3` commit.

As said yesterday, I'm interested by more information about the performances of a css class but imho it's not blocking for the bug fix in this PR.

**Comment by maukoquiroga** [comment]
Cool merci ! Add before/after and GTM

**Comment by maukoquiroga** [comment]
😃

**Pull request merged**